common/primitives/src/stateful_storage.rs

```diff
@@ -35,6 +39,8 @@ pub struct ItemizedStoragePageResponse {
 	pub msa_id: MessageSourceId,
 	///  Schema id of the
 	pub schema_id: SchemaId,
+	/// Hash of the page content
+	pub content_hash: PageHash,
 	/// Items in a page
```

Comment:
Do we want to call it content_hash ?

Reply:
I don't have strong feelings... suggestions?